lib/dartino.dart

```diff
@@ -103,6 +105,48 @@ void openDartinoSettings([_]) {
   atom.workspace.open('atom://config/packages/dartino');
 }
 
+_createNewProject([_]) async {
+  var sdk = await findSdk(null);
+  if (sdk == null) return;
```

Comment:
In this case and others, rather than all of the call sites being responsible for notifying the user, the utility methods are responsible for notifying the user if there is a problem. Specifically, `findSdk` will notify the user if the SDK cannot be found and offer to open the settings page where it can be specified.